package.json

```diff
@@ -53,18 +58,25 @@
   "license": "Apache-2.0",
   "bundlesize": [
     {
-      "path": "./dist/standalone/alloy.min.js"
+      "path": "./dist/alloy.min.js"
     }
   ],
   "dependencies": {
+    "css.escape": "^1.5.1",
+    "typescript": "^4.0.5",
```

Comment:
Does that pull typescript into the library?

Reply:
Oops, I didn't mean to add this as a dependency. Thanks for catching.